Tasks/KubernetesManifestV0/src/actions/reject.ts

```diff
@@ -9,6 +9,11 @@ export async function reject(ignoreSslErrors?: boolean) {
     const kubectl = new Kubectl(await utils.getKubectl(), TaskInputParameters.namespace, ignoreSslErrors);
 
     if (canaryDeploymentHelper.isCanaryDeploymentStrategy()) {
+        if (canaryDeploymentHelper.isTrafficSplitCanaryStrategy()) {
+            tl.debug('Redirecting traffic to stable deployment.');
+            canaryDeploymentHelper.adjustTraffic(kubectl, TaskInputParameters.manifests, 1000, 0, 0);
+        }
```

Comment:
Do we need to delete traffic service objects created for canary and baseline ?

Reply:
Not required, even after promote traffic redirected based on trafficsplit object.